src/view/pages/FarmerSearch/index.tsx

```diff
@@ -24,15 +24,17 @@ const defaultFarmerList = [{ name: 'Swalleh', phoneNumber: '1-250-234-1234', id:
                     { name: 'Simon', phoneNumber: '1-234-456-7890', id: 11 }];
 
 /** Basic model for the FarmerType object */
+// TODO: Move to store --@jinglis
 interface FarmerType {
   name: string;
   phoneNumber: string;
   id: number;
 }
 
 /** FarmerSearch OwnPropsType */
+// TODO: Make required property when moving to StorePropsType
 export interface OwnPropsType {
-  listItems?: FarmerType[];
+  farmerSearchListItems?: FarmerType[];
```

Comment:
I think `farmerList` is more appropriate, as this is coming from the store and has nothing to do with the search yet. Only after you apply the search filter to the list does it become a list of search items. We can leave this as is for now as I believe I already renamed in your fuzzy branch when I was fixing my component